src/content/self-hosted/aks.mdx

```diff
@@ -26,13 +26,7 @@ This guide will assume that your domain is registered in [AzureDNS](https://docs
 
 ### Deploy a Kubernetes cluster
 
-We recommend that you follow Azure's [cluster creation guide](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough).
-Okteto supports Kubernetes versions 1.21 through 1.23.
-
-We recommend the following specs:
-- v1.23
-- A pool with at least 3 `Standard D4` nodes
-- 250 GB per Standard SSD Managed Disk
+We recommend that you follow Azure's [cluster creation guide](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) and check our [preparation guide](self-hosted/install/preparation.mdx#deploy-a-kubernetes-cluster).
```

Comment:
The goal of these guides was to give users of every cloud-provider a one-page view of how to install Okteto. The proposed changes are requiring the installer to now read two guides instead of one.   What pain point are we trying to solve with the changes? 

Also `4CPUs, 32GB RAM and 250GB node storage` means different things on different cloud providers. Since these are cloud-provider specific guides, can we include provider-specif information, like the name of the instance we recommend using?

Reply:
The same applies to the rest of the cloud-provider guides.